src/DigitalSubscriberAppBanner.tsx

```diff
@@ -0,0 +1,332 @@
+    display: none;
+    ${until.tablet}{
+        display: block;
+        margin-left: -4px;
+        margin-top: -4px;
+        margin-bottom: 4px;
+    }
+
+`;
+
+export const infoIcon = css`
+    svg {
+        width: 60px;
+        height: 60px;
+        fill: ${palette.neutral[86]};
+    }
+    position: absolute;
+    width: 60px;
+    height: 60px;
+    left: -60px;
+    top: 10px;
+    display: none;
+    @media (min-width: 1430px){
+        display: block;
+    }
+`;
+
+export const storeIcon = css`
+    height: 10px;
+    svg {
+        height: 30px;
+        width: auto;
+        margin-top: ${space[2]}px;
+        padding-right: ${space[2]}px;
+    }
+    display: inline-block;
+    vertical-align: -37.5%;
+    
+`;
+
+export const DigitalSubscriberAppBanner: React.FC<Props> = ({
+  message,
+  onButtonClick,
+  firstName
+}: Props) => {
+    const [showBanner, setShowBanner] = useState(true);
+
+    const onCloseClick = (evt: React.MouseEvent<HTMLButtonElement, MouseEvent>, buttonId: string): void => {
+        evt.preventDefault();
+        onButtonClick(buttonId);
+        setShowBanner(false);
+    };
+    
+    if (!showBanner) return null;
+    else return (
+        <div css={wrapper}>
+            <div css={contentContainer}>
+                <div css={topLeftComponent}>
+                    <div css={infoIcon}>
+                        <SvgInfo />
+                    </div>
+                    <div css={heading}><span css={smallInfoIcon}>
+                        <SvgInfo />
+                    </span> A note to our digital subscribers</div>
+                    <p css={paragraph}>
+                        {firstName ? "Hi " + firstName : "Hello"}, did you know that as a Guardian digital subscriber you can enjoy an enhanced experience of our quality, independent journalism on all your devices, including The Guardian Live app.
+                        <br/>
+                        <strong css={strongSmallSpacer}>
+                            Search for "Guardian live news"
+                        </strong>
+                        <span css={storeIcon}>
+                            <AppStore />
+                            <PlayStore />
+                        </span>
+                    </p>
+                    <Button onClick={e => onCloseClick(e, '0')} css={primaryButton}>Ok, got it</Button>
+                    <Button onClick={e => onCloseClick(e, '1')} css={secondaryButton} priority="subdued">I'm not interested</Button>
+                </div>
+                <div css={bottomRightComponent}>
+                    <div css={packShot}>
+                        <img
+                            src="https://via.placeholder.com/400x300"
+                            alt=""
+                        />
+                    </div>
+                    <div css={iconPanel}>
+                        <button
+                            aria-label="Close"
+                            onClick={e => onCloseClick(e, '1')}
```

Comment:
What would be the impact of giving this button ID '2'?

Reply:
I'm not sure actually. '0' maps to 'Button 1' in the Braze interface and '1' maps to 'Button 2'. I'll give it a go and see.

Reply:
It seems to be coming through as a 'Body Click' when the index is 2. My guess is 0 -> 'Button 1', 1 -> 'Button 2', everything else -> 'Body Click'